addons/xpaas-templates/xpaas-templates.addon

```diff
@@ -0,0 +1,96 @@
+# Name: xpaas
+# Description: Imports xPaaS templates
+# Url: https://github.com/jboss-openshift/application-templates/tree/ose-v1.3.6
+
+oc create -f jboss-image-streams.json -n openshift
```

Comment:
Apply should be safer, in case an addon is applied when a template has already been created. Maybe in the update process, in case you are applying an update to the addon